python/ray/tests/test_logging.py

```diff
@@ -377,6 +379,10 @@ def f():
     ), f"Python stack trace not found in stderr: {stderr}"
 
 
+@pytest.mark.skipif(
+    sys.platform == "win32" or sys.platform == "darwin",
+    reason="TODO(simon): Failing on Windows and OSX.",
```

Comment:
@simon-mo I'll leave it to you to fix this: seems java package is not built for windows and osx.
```
(raylet) [2022-03-09 00:41:15,009 C 66657 3215559] (raylet) worker_pool.cc:1316:  Check failed: state != states_by_lang_.end() Required Language isn't supported: JAVA
```